CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 ### Fixed
 
 - [#2501](https://github.com/thanos-io/thanos/pull/2501) Query: gracefully handle additional fields in `SeriesResponse` protobuf message that may be added in the future.
+- [#2525](https://github.com/thanos-io/thanos/pull/2525) Query: Added logging for dns resolution error in the `Query` component.
```

Comment:
So fixed or added? (; You put this in `Fixed` section , but we said "Added" here... ^^

I think it's fixed as we fix issue

Reply:
Sure, some sloppy grammar from my side :P